Autopilot/Inc/ultrasonic.hpp

```diff
@@ -0,0 +1,34 @@
+/**
+ * Ultrasonic Sensor Functions
+ * Author: Dhruv Upadhyay, February 2021
+ */
+
+ #ifndef ULTRASONIC_HPP
+ #define ULTRASONIC_HPP
+
+ #include <stdint.h>
+
+typedef struct {
+    float distance;
+    bool isDataNew; // Holds whether the sensor data has been updated or not
+    uint8_t status; // -1 for trig not sent and echo not received, 0 for trig sent, echo not received, and 1 for echo received
+    float time; // represents the time taken between trig being sent and echo being received
+    const int SOUND_SPEED = 343; // In meters per second, used to calculate distance based on time taken to send and receive ping
```

Comment:
Is there a reason that the const is inside the struct as opposed to just living in the class?

Reply:
No it should be outside of the struct, mb